sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.net8.0.cs

```diff
@@ -6831,6 +6967,7 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         private readonly int _dummyPrimitive;
         public NetworkApiVersion(string value) { throw null; }
         public static Azure.ResourceManager.Compute.Models.NetworkApiVersion TwoThousandTwenty1101 { get { throw null; } }
+        public static Azure.ResourceManager.Compute.Models.NetworkApiVersion TwoThousandTwentyTwo1101 { get { throw null; } }
```

Comment:
LOL these are really bad names.
I think these are the api-versions for network RP, right?
Then let us fix it via customization code like this:
```
public readonly partial struct NetworkApiVersion
{
	[CodeGenMember("TwoThousandTwentyTwo1101")]
	public static NetworkApiVersion v2022_11_01 {get;} = new NetworkApiVersion(v2022_11_01Value);
}
```

Reply:
added NetworkApiVersion.cs in the custom folder accordingly

Reply:
If I create a custom like this: 
![image](https://github.com/user-attachments/assets/f224c5ce-d110-4c22-8048-273dde54f432)

Then next time I generate code, the original generated/models/networkApiVersion.cs file gets updated like this. 
So then I get build error on the custom file after that

Reply:
you just need to fix it by changing your XXXValue to those now exist
You might notice in my example, I am already writing in that way.

Reply:
I realized it was an existing enum. So I added originals as alias to preserve the originals
https://github.com/Azure/azure-sdk-for-net/pull/49063/commits/6ff928e1a679c36ed39d196db2ac7ceea74dfe8c